guide/12-enrich-data-with-thematic-information/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -11,139 +11,576 @@
     "# Part 1 - Introduction to GeoEnrichment"
```

Comment:
<p>GeoEnrichment makes any location data intelligent by providing facts about the location. In this part of the Geoenrichment guide series, you have seen a high-level example of how <code>arcgis.geoenrichment.Country</code> can be used to <code>enrich</code> a dataset with various socio-demographic features. You also saw an introduction of the different ways in which data can be enriched. In the subsequent pages, you will learn about: </p><ol><li>Enriching Study Areas (explains where to enrich)</li><li>Exploring Named Statistical Areas (explains where to enrich continued)</li><li>Enriching Data Collections and Spatially Enabled Dataframe (explains what datasets/variables to enrich with)</li><li>Generating Reports</li><li>Standard Geography Queries</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>